app/models/manageiq/providers/ibm_cloud/container_manager.rb

```diff
@@ -0,0 +1,201 @@
+class ManageIQ::Providers::IbmCloud::ContainerManager < ManageIQ::Providers::Kubernetes::ContainerManager
```

Comment:
@nasark can you add `ManageIQ::Providers::Kubernetes::ContainerManager.include(ActsAsStiLeafClass)` above this like https://github.com/ManageIQ/manageiq-providers-azure/blob/master/app/models/manageiq/providers/azure/container_manager.rb#L1 ?